src/routes/lists/__snapshots__/Lists.spec.jsx.snap

```diff
@@ -294,7 +294,7 @@ exports[`Lists renders Lists when data retrieval is complete 1`] = `
                   <a
                     class="p-0 me-3 list-button-enabled"
                     data-test-id="incomplete-list-share"
-                    href="/lists/id2/users_lists"
+                    href="/id2/users_lists"
```

Comment:
i don't get this... in e2e testing the links were "/lists/id2/user_lists". we need to be sure about this. it may cause problems

Reply:
e2e testing looks good. just need to keep an eye on this